web/packages/teleterm/src/services/tshd/testHelpers.ts

```diff
@@ -94,7 +94,7 @@ export const makeRootCluster = (
   name: 'teleport-local',
   connected: true,
   leaf: false,
-  proxyHost: 'teleport-local:3080',
+  proxyHost: 'teleport-local.com:3080',
```

Comment:
This is purely so that stories reflect the reality more closely. I doubt that the majority of people have a cluster set up as a TLD.